### PR TITLE
added getters for paths and validOSes

### PR DESCRIPTION
**Pull request by veecue**
Adjustments for a program as discussed in #13 to work

### CODE REVIEW

File: paths.go [hunk 1]
```diff
@@ -86,8 +86,11 @@ func (p *Paths) initAstilectronDirectory() {
 	}
 }
 
-// initElectronDownloadSrc initializes the electron download source path
-func (p *Paths) initElectronDownloadSrc(os, arch string) {
+func AstilectronDownloadSrc() string {
```
Comment (asticode) [inline paths.go]: Can you add a comment on top of this function of the form `// AstilectronDownloadSrc`?

File: paths.go [hunk 2]
```diff
@@ -86,8 +86,11 @@ func (p *Paths) initAstilectronDirectory() {
...
+	return fmt.Sprintf("https://github.com/asticode/astilectron/archive/v%s.zip", VersionAstilectron)
+}
+
+func ElectronDownloadSrc(os, arch string) string {
```
Comment (asticode) [inline paths.go]: Can you add a comment on top of this function of the form `// ElectronDownloadSrc`?

File: astilectron.go [hunk 3]
```diff
@@ -118,6 +118,10 @@ func validateOS(os string) error {
 	return nil
 }
 
+func ValidOSes() []string {
```
Comment (asticode) [inline astilectron.go]: I feel like exporting `validOSes` makes more sense. What do you think?

Comment (veecue) [inline astilectron.go]: If you are willing to take the risk of somebody modifying the array from the outside, I'll be happy to do it instead

Comment (asticode) [inline astilectron.go]: That's fair. Could you just add a comment of the form `// ValidOSes`?

**Pull request merged**